### PR TITLE
Add jupytext

### PR DESCRIPTION
**Pull request by melissawm**
Also sets up initial sync configuration between markdown and python files; adds download button to all tutorials; adds pre-commit hook to set up automatic check of sync'd md/py files.

I am happy to undo some of the whitespace changes if this is too much, but since jupytext can be very picky around new lines, I had to change some manually and figured having a check for this might help too, so a few extra files were touched.

Happy to take other approaches if preferred. Thanks!

### CODE REVIEW

**Comment by melissawm** [comment]
Button looks like this (with the primary blue highlight on hover):
Screenshot 2023-12-13 at 10 20 13 PM


Screenshot 2023-12-13 at 10 21 01 PM

**Review by dgmccart — APPROVED** [review]

**Comment by melissawm** [comment]
Yes, there are steps. Let me write a guide and you can try it out and see if the instructions are enough.

File: docs/tutorials/chunked.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     formats: md,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.16.0
+# ---
```
Comment (aliddell) [inline docs/tutorials/chunked.py:11]: Is there any way to skip this?

**Review by aliddell — APPROVED** [review]
Looks good to me, but I'm not a fan of

```
# ---
# jupyter:
#   jupytext:
#     cell_metadata_filter: -all
#     formats: md,py
#     text_representation:
#       extension: .py
#       format_name: light
#       format_version: '1.5'
#       jupytext_version: 1.16.0
# ---
```

at the top of every Python script. That shouldn't hold this up any longer though.

**Comment by melissawm** [comment]
yeah - this is the metadata that keeps things in sync, unfortunately. So the other approach would be to not keep things in sync this way, but just re-run the converter on push, instead of on commit. That would mean this logic would move from the pre-commit hook to CI. I can do it as a proof of concept and we can go with the option that looks less intrusive. Wdyt?

**Comment by aliddell** [comment]
> yeah - this is the metadata that keeps things in sync, unfortunately. So the other approach would be to not keep things in sync this way, but just re-run the converter on push, instead of on commit. That would mean this logic would move from the pre-commit hook to CI. I can do it as a proof of concept and we can go with the option that looks less intrusive. Wdyt?

Sure, by all means.

**Comment by melissawm** [comment]
Alright - here's the second option. It's a lot cleaner so I assume this is what we'll do. I've kept both commits for comparison here so ideally we can drop one of them before merging.

If you want to see the results I've tested on my fork here: https://github.com/melissawm/acquire-docs/actions/runs/7338869048/job/19982085030#step:5:1

And the rendered website: https://melissawm.github.io/acquire-docs/tutorials/trigger/#acquire-data

**Comment by melissawm** [comment]
Ok folks, this is updated and should be good to go - pending on the design of course! If you prefer different buttons or colors let me know. Cheers!

**Comment by aliddell** [comment]
SHIP IT

**Pull request merged**